shell/platform/tizen/tizen_window.h

```diff
@@ -65,6 +65,10 @@ class TizenWindow {
   // This is a temporary implementation that is only used by the window channel.
   virtual void OnGeometryChanged(Geometry geometry) = 0;
 
+  TizenInputMethodContext* input_method_context() {
```

Comment:
How about `GetInputMethodConetext()` or `GetInputMethodConetextPtr()`?

Reply:
we don't use xxPtr postfix in method name. we name get/setter according to [this](https://google.github.io/styleguide/cppguide.html#Function_Names) 

In particular, for getters that only return references from unique pointers, we are using this format widely.

Reply:
oh.. ok,,